.github/workflows/test-chart.yml

```diff
@@ -0,0 +1,31 @@
+name: Testsuite
```

Comment:
@ryanemerson seems to workflow need to be in the repo before. I never setup GitHub actions workflow before. Can you help me with it?

Reply:
I think the file may have to be in the repo before it can be used, which would explain why it works on your repository. I think we should merge this PR when everything else is ready and if the GH action still doesn't work we can investigate further.

Reply:
Thanks.